recipe/meta.yaml

```diff
@@ -1,64 +1,68 @@
 {% set name = "frictionless" %}
-{% set version = "5.5.6" %}
+{% set version = "4.40.11" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: a3537634989ae2aa294d8031dc70630bf904191b1de05570097b87c706b38146
+  sha256: e7d83d82cd3273820c74ac715e8d78285697f1eceda49a2417a72f839420d42e
 
 build:
   number: 0
   entry_points:
     - frictionless = frictionless.__main__:program
-  script: {{ PYTHON }} -m pip install . --no-deps -vv
-  noarch: python
+  script: {{ PYTHON }} -m pip install . --no-deps -vv --no-build-isolation
 
 requirements:
   host:
     - python
     - pip
+    - wheel
   run:
-    - python >=3.8
+    - python
     - petl >=1.6
-    - marko >=1.0
-    - attrs >=21.3
-    - jinja2 >=3.0
+    - marko >=1.0,<2
```

Comment:
Are there incompatibility issues with v2+?

Reply:
There are breaking changes in 2.0